### ISSUE TITLE
[Enhancement] Show User who started tasksequence on "Execution History"

### ISSUE THREAD

**Issue by horst789github** [rzander/sccmclictr#149]
can you please add another column that shows which user, when, started the tasksequence/program?

**Comment by rzander** [comment]
from where should I get this Information ?

**Comment by horst789github** [comment]
:D good question

**Comment by SCCMAdmin** [comment]
@horst789github why would this be handy? Is this for Software Center invoked items?

I've seen such information in a log file before, maybe one of the sc logs. @rzander I guess you are trying to work out is this info saved in WMI somewhere?

**Comment by horst789github** [comment]
@SCCMAdmin yes for Software Center items or general task sequence triggerd over powershell... we have the problem, that a user say: my server is automaticly patching/install software... why/who has started this... and this feature will very helpful

**Comment by rzander** [comment]
I have two concerns on this request:
1) Is this Information stored somewhere (not in a log-file)... and even if it's somewhere...
2) I do net see the benefit (use case) behind this... A user is able to trigger what the Admin allows to be triggered... Otherwise hide these risky Items from SWCenter ?!

**Comment by horst789github** [comment]
1. i dont know
2. other admins start this, not user...

**Comment by horst789github** [comment]
ok i found where the info is stored...
ccm\logs\SCClient_[Domain]@[Username]_[number].log file and there is example:
Performing action: Install on application [TASK SEQUENCE Name]
Installing programm * from package [ID]
